src/index.html

```diff
@@ -4,19 +4,112 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <link
+      rel="stylesheet"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
     />
-    <title>Moyo header</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
-      href="./style.css"
     />
+    <title>MOYO</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="MOYO"
+        />
+      </a>
+
+      <nav>
+        <ul class="list">
+          <li class="link">
+            <a
```

Comment:
The `href` attribute is set to `#`, which is a placeholder. Ensure that the actual URLs are updated when the site is deployed to make the links functional.